hw/ip/clkmgr/doc/_index.md

```diff
@@ -226,12 +227,24 @@ There is currently no support in hardware to dynamcially synchronize a threshold
 
 The table below summarises the valid modes and the settings required.
 
-| Mode                         | `lc_ctrl_clk_byp_req`  | `extclk_ctrl.sel` | `extclk_ctrl.low_speed_sel` | life cycle state        |
+| Mode                         | `lc_ctrl_clk_byp_req`  | `extclk_ctrl.sel` | `extclk_ctrl.hi_speed_sel`  | life cycle state        |
 | -------------                | ---------------------  | ----------------- | ----------------------------| ----------------------- |
 | Life cycle transition        | `lc_ctrl_pkg::On`      | Don't care        | Don't care                  | Controlled by `lc_ctrl` |
 | Internal Clocks              | `lc_ctrl_pkg::Off`     | `kMultiBit4False` | Don't care                  | All                     |
-| Software external high speed | `lc_ctrl_pkg::Off`     | `kMultiBit4True`  | `kMultiBit4False`           | TEST_UNLOCKED, RMA      |
-| Software external low speed  | `lc_ctrl_pkg::Off`     | `kMultiBit4True`  | `kMultiBit4True`            | TEST_UNLOCKED, RMA      |
+| Software external high speed | `lc_ctrl_pkg::Off`     | `kMultiBit4True`  | `kMultiBit4True`            | TEST_UNLOCKED, RMA      |
+| Software external low speed  | `lc_ctrl_pkg::Off`     | `kMultiBit4True`  | `kMultiBit4False`           | TEST_UNLOCKED, RMA      |
+
+The table below summarizes the frequencies in each mode.
+This table assumes that the internal clock source is 96MHz.
+This table also assumes that high speed external clock is 96MHz, while low speed external clock is 48MHz.
+
+| Mode                         | div_1_clock   | div_2_clock     | div_4_clock  |
```

Comment:
@matutem let me know if this is clearer.
An example probably helps.

Reply:
ah it could in theory. But the problem is if it accepted 100MHz, it would imply that one of the divided down clocks would be 50MHz, and that is faster than timing closure.

So unfortunately when using external clock it kind of needs to pick the lowest common denominator.